src/praisonai/tests/integration/gateway/test_magic_link_flow.py

```
@@ -0,0 +1,333 @@
+
+
+@pytest.fixture
+def magic_store(temp_storage):
+    """Provide a MagicLinkStore for testing."""
+    return MagicLinkStore(storage_path=temp_storage / "magic-links.json")
+
+
+@pytest.fixture
+def auth_manager():
+    """Provide a CookieAuthManager for testing."""
+    return CookieAuthManager(secret_key="test-secret-key")
```

Ruff (0.15.10)

[error] Possible hardcoded password assigned to argument: "secret_key"

(S106)